### DISCUSSION TITLE
Time series load flow calculation

### DISCUSSION THREAD

**Discussion by fapermata8** [SanPen/GridCal#283] [Q&A] [answered]
Hello, i was using GridCal 5.1.8 to perform load flow calculation. I have input generator and load data using GUI interface. On the result i can understand if generator that connected to slack bus is have different value with my setting. my question is why other generator that i set Zero on the input generator time series have result Active and Reactive power that more than the limit? thank you for you help
![Screenshot 2024-05-26 060951](https://github.com/SanPen/GridCal/assets/148619924/fd7817f1-314f-460e-9bf4-a57f3135b04b)
![Screenshot 2024-05-26 060921](https://github.com/SanPen/GridCal/assets/148619924/78ddf445-e7a1-497c-9184-faa35560d849)

**Comment by SanPen** [comment, marked as answer]
Hi!

I believe the discrepancy is due to fact that the reactive power in generation devices is calculated if the device is_controlled property is true.

This is, gerators make buses PV and in PV buses, Q is calculated regardless of what Q or pf you specify...if is_controlled = true, else the bus is PQ and the reactiva power estimate remains.

Then, where does Q come from at the inputs analysis? An initial Q estimate is generated with the generator power factor (pf) but that estimate is corrected by the power flow algorithm if is_controlled = true.

I hope this helps

**Reply by fapermata8** [reply]
Thank you for the explanation,It  is solved by changing is_controlled = False. 
![Screenshot 2024-05-27 131200](https://github.com/SanPen/GridCal/assets/148619924/43bafacd-f854-4658-a4b5-20eaee2c358b)

**Reply by SanPen** [reply]
nice!
